docs/conf.py

```diff
@@ -63,7 +63,7 @@
 # built documents.
 #
 # The full version, including alpha/beta/rc tags.
-release = '0.8.0.dev1'
+release = '0.8.0.dev0'
```

Comment:
merge problem?

Reply:
Not a merge problem. The issue was that the build_release script didn't update the nifake version when it updates. I fixed the script and manually changed nifake's version to 0.8.0.dev1.